modules/ocf_keyserver/files/sks-keyserver.service

```diff
@@ -11,6 +11,7 @@ ExecStart=/usr/bin/docker run --rm \
     -p 127.0.0.1:11372:11371/tcp \
     --name sks-keyserver \
     zhusj/sks
+CPUQuota=50%
```

Comment:
Have you verified that this quota will be forwarded to the Docker container?

Reply:
`cgroups` are hierarchical so I don't imagine this will be an issue. Once I test it I will leave a follow up comment though.

Reply:
The `docker run` command is just sending commands to the Docker daemon over the network, your container's processes are not a descendant of `docker run`. I would be surprised if this works.

Reply:
I see. Thanks for the comments. I think the newest commit addresses this.